test/test_interactive.py

```diff
@@ -334,6 +334,20 @@ def test_audio_worklet_2x_hard_pan_io(self):
     shutil.copy(test_file('webaudio/audio_files/emscripten-bass-mono.mp3'), 'audio_files/')
     self.btest_exit('webaudio/audioworklet_2x_in_hard_pan.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS'])
 
+  # Same as 'test_audio_worklet_2x_stereo_io' but as wasm64 with the worklet stack starting 2GB in (tests for unsigned shifts in the memory offsets)
+  def test_audio_worklet_stereo_io_2gb(self):
+    os.mkdir('audio_files')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-beat.mp3'), 'audio_files/')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-bass.mp3'), 'audio_files/')
+    self.btest_exit('webaudio/audioworklet_in_out_stereo.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-sMEMORY64', '-sINITIAL_MEMORY=2164260864', '-DTEST_OFFSET_GB=2'])
```

Comment:
The `_2gb` test are normally testing the high 2gb of wasm32 mode.   In this mode we need to use `>> 0` rather than `>>> 0` for example.

Reply:
I'll add a separate wasm32 test (where `GLOBAL_BASE` won't cause `malloc` issues with trying to claim 2GB).

Reply:
I made the 2GB interactive test run with wasm32, so now tests are for `>>`, `>>>` and `/ 4` via `getHeapOffset()`.